learn/update_and_migration/versioning.md

```diff
@@ -0,0 +1,37 @@
+# Versioning policy
+
+Meilisearch releases contain performance improvements, bug fixes, and new features, but may not always maintain full compatibility between Meilisearch versions.
```

Comment:
The habit of writing intro/topic sentences/paragraphs is a difficult one to unlearn, eh? 😁 

```suggestion
```